webapps-schema/src/main/resources/schema/elasticsearch/create/index/identity-groups.json

```
@@ -11,8 +11,14 @@
       "name": {
         "type": "keyword"
       },
-      "assignedMemberKeys": {
+      "entityKey": {
```

```suggestion
      "memberKey": {
```
That would make it a bit clearer that it's a property of the child.